<ui version="4.0" >
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="BufferWidget" native="1" name="bufferWidget" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuNetworks" >
     <property name="title" >
      <string>&amp;Networks</string>
     </property>
     <addaction name="separator" />
    </widget>
    <addaction name="actionConnectCore" />
    <addaction name="actionDisconnectCore" />
    <addaction name="separator" />
    <addaction name="menuNetworks" />
    <addaction name="separator" />
    <addaction name="actionJoinChannel" />
    <addaction name="separator" />
    <addaction name="actionSetAwayGlobally" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSettingsDlg" />
   </widget>
   <widget class="QMenu" name="menuViews" >
    <property name="title" >
     <string>&amp;Views</string>
    </property>
    <addaction name="actionManageViews" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutQuassel" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menuDebug" >
    <property name="title" >
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionDebug_Console" />
   </widget>
   <widget class="QMenu" name="menuCore" >
    <property name="title" >
     <string>&amp;Core</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuViews" />
   <addaction name="menuCore" />
   <addaction name="menuSettings" />
   <addaction name="menuDebug" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionNetworkList" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Network List...</string>
   </property>
   <property name="shortcut" >
    <string>F2</string>
   </property>
  </action>
  <action name="actionQuickConnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Quick &amp;Connect...</string>
   </property>
  </action>
  <action name="actionDisconnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionReconnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionJoinChannel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Join Channel...</string>
   </property>
  </action>
  <action name="actionSetAwayGlobally" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Set Away globally</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="../../icons/icons.qrc" >:/22x22/actions/oxygen/22x22/actions/application-exit.png</iconset>
   </property>
   <property name="text" >
    <string>Quit...</string>
   </property>
  </action>
  <action name="actionEditIdentities" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Edit Identities...</string>
   </property>
  </action>
  <action name="actionSettingsDlg" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Configure Quassel...</string>
   </property>
   <property name="shortcut" >
    <string>F7</string>
   </property>
  </action>
  <action name="actionManageViews" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Manage Views...</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAboutQuassel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../icons/icons.qrc" >:/icons/quassel-icon.png</iconset>
   </property>
   <property name="text" >
    <string>About Quassel IRC...</string>
   </property>
  </action>
  <action name="actionImportBacklog" >
   <property name="text" >
    <string>Import Backlog</string>
   </property>
  </action>
  <action name="actionConnect_to_Core" >
   <property name="text" >
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect_2" >
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionInternal" >
   <property name="text" >
    <string>Internal</string>
   </property>
  </action>
  <action name="actionDisconnectCore" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Disconnect from Core</string>
   </property>
  </action>
  <action name="actionConnectCore" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Connect to Core...</string>
   </property>
  </action>
  <action name="actionDebug_Console" >
   <property name="text" >
    <string>Debug &amp;Console</string>
   </property>
  </action>
  <action name="actionDisconnectNet" >
   <property name="text" >
    <string>Disconnect from Network...</string>
   </property>
  </action>
  <action name="actionEditNetworks" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Edit Networks...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BufferWidget</class>
   <extends>QWidget</extends>
   <header>bufferwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
